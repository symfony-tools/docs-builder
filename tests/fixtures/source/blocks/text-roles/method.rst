
:method:`Symfony\\Component\\HttpFoundation\\RequestStack::getCurrentRequest`
