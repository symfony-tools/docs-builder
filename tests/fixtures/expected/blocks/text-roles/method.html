<p><a href="https://github.com/symfony/symfony/blob/4.0/src/Symfony/Component/HttpFoundation/RequestStack.php#method_getCurrentRequest" class="reference external" title="Symfony\Component\HttpFoundation\RequestStack::getCurrentRequest()" rel="external noopener noreferrer" target="_blank">getCurrentRequest()</a></p>
